src/fetcher.ts

```diff
@@ -85,6 +86,15 @@ export function createFetchComponent(defaultOptions?: FetcherOptions): IFetchCom
       throw new Error(`Failed to fetch ${url}. Got status ${response?.status}. Response was '${responseText}'`)
     }
 
+    if (!isUsingNode() && !!response) {
+      Object.defineProperty(response, 'buffer', {
+        value: async function (): Promise<Buffer> {
+          return Buffer.from(await response!.arrayBuffer())
```

Comment:
is it required the assertion `response!`?

Reply:
I think so, because of the abort signal. It is just a safe-check to prevent setting the buffer property to `undefined`.